src/cljs/ataru/virkailija/application/view.cljs

```diff
@@ -717,15 +717,34 @@
          :else "Tuntematon"))
 
 (defn to-event-row
-  [time-str caption]
-  [:div.application-handling__event-row
-   [:span.application-handling__event-timestamp time-str]
-   [:span.application-handling__event-caption caption]])
+  [time-str caption event]
+  (let [modify-event? (util/modify-event? event)]
+    [:div.application-handling__event-row
+     [:span.application-handling__event-timestamp time-str]
+     [:span.application-handling__event-caption
+      {:on-click (when modify-event? #(dispatch [:application/toggle-event-expanded (:id event)]))}
+      caption
+      (when (and modify-event?
+                 (some #{(:id event)} @(subscribe [:application/expanded-event-ids])))
+        [:ul.application-handling--event-row-details
+         (for [[key field] @(subscribe [:application/changes-made-for-event (:id event)])]
+           ^{:key (str "event-list-for" key)}
```

Comment:
Mielipidehomma, mut mun mielestä ois siistimpää jos tää `:key` ois ton `li`:n attribuuttimapin sisällä kerta sellanen jo on (eli tossa `on-click`in yhteydessä)

Reply:
👍